regolith/helpers/l_grantshelper.py

```diff
@@ -60,14 +63,35 @@ def construct_global_ctx(self):
     def sout(self):
         rc = self.rc
         grants = []
+        desired_date = None
```

Comment:
will something like this work?:
```
if rc.date:
  now = rc.date
else:
  now = datetime.today().date()
```
then the rest as before?  I think we only wanted to change it so that we could pass in a date to ``is_current(grant, date=now)``.

Reply:
I designed the lister so that you can pass in both a -c and -d DATE flag (you may want all current grants but also all grants from 2 years ago). I do not think changing the code like that would still ensure you can pass both flags and get the correct output.

Reply:
What happens if I pass in -c (the current date) and -d DATE for a specific date. I still want to check if a grant is current, but also current by the passed in date.  With that variable now I may have a date I passed by I may also still want to check for current grants, so assigning it to one value in that situation may be bad

Reply:
So the lister should not be able to take both the -c flag and the -d flag at once?